programs/stake/src/stake_state.rs

```diff
@@ -897,11 +912,7 @@ fn validate_split_amount(
     // This must handle:
     // 1. The destination account having a different rent exempt reserve due to data size changes
     // 2. The destination account being prefunded, which would lower the minimum split amount
-    let destination_rent_exempt_reserve = calculate_split_rent_exempt_reserve(
-        source_meta.rent_exempt_reserve,
-        source_account.data_len()? as u64,
-        destination_account.data_len()? as u64,
-    );
+    let destination_rent_exempt_reserve = rent.minimum_balance(destination_account.data_len()?);
```

Comment:
This is the meat of this set, right?

technically we're breaking consensus here, but I really have my doubts we'll be modifying `lamports_per_byte_year` before this is fully deployed

Reply:
I'll just add a feature just in case because it doesn't hurt at all

Reply:
Feature added and I added back the original code path for now until the feature is activated and it can be removed